end-to-end-tests/get-blur-areas.sh

```diff
@@ -0,0 +1,31 @@
+# Makes a POST request with an image as parameter and compares the response with a file's content.
+# First parameter is the host to make the request to.
+# Second parameter is image's name.
+# Third parameter is the name of the file containing the expected output.
+check_image() {
```

Comment:
It's better to check that the function takes exactly 3 arguments in the beginning.

Reply:
Right! I added a check, thank you